<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>
        Pedro Fontanarrosa's CV
    </title>
    <link rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/5.5.1/github-markdown-light.min.css"
        integrity="sha512-Pmhg2i/F7+5+7SsdoUqKeH7UAZoVMYb1sxGOoJ0jWXAEHP0XV2H4CITyK267eHWp2jpj7rtqWNkmEOw1tNyYpg=="
        crossorigin="anonymous" referrerpolicy="no-referrer" />
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.12/dist/katex.min.css" integrity="sha384-PDbUeNCuE6bOPudPOgFyIUEy3UJawJVwr3XlGO90FIuf5qNIoTLSgOJo/dC2ZXV/" crossorigin="anonymous">

    <!-- The loading of KaTeX is deferred to speed up page rendering -->
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.12/dist/katex.min.js" integrity="sha384-VkqWq8xtm5YQk1BBXczQ8/Sx+DlCzF8cuS43bZwmtVXzRFtyLTqTCdP7MKmKo+KN" crossorigin="anonymous"></script>

    <!-- To automatically render math in text elements, include the auto-render extension: -->
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.12/dist/contrib/auto-render.min.js" integrity="sha384-hCXGrW6PitJEwbkoStFjeJxv+fSOOQKOPbJxSfM6G5sWZjAyWhXiTIIAmQqnlLlh" crossorigin="anonymous"
        onload="renderMathInElement(document.body, {delimiters: [{ left: '$$',  right: '$$',  display: false }]});"></script>
    <style>
        .markdown-body {
            box-sizing: border-box;
            min-width: 200px;
            max-width: 980px;
            margin: 0 auto;
            padding: 45px;
        }

        @media (max-width: 767px) {
            .markdown-body {
                padding: 15px;
            }
        }
    </style>
</head>

<body>
    <article class="markdown-body">
        <h1>Pedro Fontanarrosa's CV</h1>
        <ul>
        <li>Phone: +44 7564 594770</li>
        <li>Email: <a href="mailto:pfontanarrosa@gmail.com">pfontanarrosa@gmail.com</a></li>
        <li>Location: London, England, GB</li>
        <li>Website: <a href="https://fontanapink.github.io/Resume-PedroFontanarrosa/">fontanapink.github.ioResume-PedroFontanarrosa</a></li>
        <li>GitHub: <a href="https://github.com/Fontanapink">Fontanapink</a></li>
        <li>ORCID: <a href="https://orcid.org/0000-0002-0535-2684">0000-0002-0535-2684</a></li>
        <li>Google Scholar: <a href="https://scholar.google.com/citations?user=Pedro Fontanarrosa">Pedro Fontanarrosa</a></li>
        <li>ResearchGate: <a href="https://researchgate.net/profile/Pedro Fontanarrosa">Pedro Fontanarrosa</a></li>
        <li>LinkedIn: <a href="https://linkedin.com/in/Pedro Fontanarrosa">Pedro Fontanarrosa</a></li>
        </ul>
        <h1>Profile</h1>
        <p>Broadly skilled researcher with a strong foundation in computational/software engineering and biological sciences.</p>
        <p>Extensive experience developing Genetic Design Automation (GDA) tools and mathematical models for genetic circuit design during Master's and PhD research.</p>
        <p>Currently expanding into advanced machine learning algorithm development at UCL—designing custom Gaussian process regression kernels, applying Bayesian inference with VAR regression, and developing physics-informed neural networks.</p>
        <p>Excels in remote, multidisciplinary collaborations.</p>
        <h1>Education</h1>
        <h2>University of Utah, Ph.D. in Biomedical Engineering (Synthetic Biology Track)</h2>
        <ul>
        <li>Aug 2019 – Aug 2022</li>
        <li>Salt Lake City, UT, USA</li>
        <li>GPA: 3.8/4.0</li>
        <li>Courses: []</li>
        </ul>
        <h2>University of Utah, Master in Bioengineering (Synthetic Biology Track)</h2>
        <ul>
        <li>Aug 2017 – Aug 2019</li>
        <li>Salt Lake City, UT, USA</li>
        <li>GPA: 3.67/4.0</li>
        <li>Courses: []</li>
        </ul>
        <h2>University of Buenos Aires, Licentiate in Biological Sciences (Genetics, Evolution and Ecology)</h2>
        <ul>
        <li>Jan 2007 – Jan 2014</li>
        <li>Buenos Aires, Argentina</li>
        <li>GPA: 8.39/10.00</li>
        <li>Courses: []</li>
        </ul>
        <h1>Experience</h1>
        <h2>Computational Systems and Synthetic Biology Lab, University College London, Postdoctoral Researcher</h2>
        <ul>
        <li>June 2023 – present</li>
        <li>London, UK</li>
        <li>Developed robust Python packages for genetic design automation and modeling inference</li>
        <li>Created and adapted CI/CD pipelines for scalable computational biology workflows</li>
        <li>Applied and advanced machine learning (GP, VAR, NN) and Bayesian optimization algorithms to enhance predictive modeling and system design</li>
        </ul>
        <h2>Genetic Logic Lab, University of Boulder, Postdoctoral Researcher</h2>
        <ul>
        <li>Aug 2022 – June 2023</li>
        <li>Boulder, CO, USA</li>
        <li>Enhanced iBioSim functionalities for genetic circuit modeling</li>
        <li>Developed robust mathematical models for genetic design</li>
        <li>Collaborated with international research teams using cloud-based tools</li>
        </ul>
        <h2>University of Utah, Research Assistant (PhD Researcher)</h2>
        <ul>
        <li>Aug 2019 – Aug 2022</li>
        <li>Salt Lake City, UT, USA</li>
        <li>Simulated genetic regulatory networks using stochastic modeling, forecasting gene expression levels with 95% accuracy and guiding subsequent experimental validation by other team members.</li>
        <li>Guided experimental research through simulation-based predictions</li>
        <li>Collaborated on interdisciplinary synthetic biology projects</li>
        </ul>
        <h2>COMBINE Standards, SBOL Editor</h2>
        <ul>
        <li>Jan 2019 – Jan 2022</li>
        <li>Online</li>
        <li>Led weekly meetings to coordinate SBOL standard updates</li>
        <li>Facilitated community-driven revisions and improvements to SBOL</li>
        <li>Enhanced interoperability in synthetic biology data exchange</li>
        </ul>
        <h2>University of Utah, Research Assistant (Master's Researcher)</h2>
        <ul>
        <li>Aug 2017 – Aug 2019</li>
        <li>Salt Lake City, UT, USA</li>
        <li>Contributed to preliminary genetic circuit modeling tools</li>
        <li>Supported modeling and simulation of genetic regulatory systems</li>
        <li>Collaborated on interdisciplinary research initiatives</li>
        </ul>
        <h2>Northlands School, Highschool Chemistry Teacher</h2>
        <ul>
        <li>Jan 2015 – Jan 2017</li>
        <li>Buenos Aires, Argentina</li>
        <li>Developed lesson plans that enhanced student engagement</li>
        <li>Mentored students in chemical research projects</li>
        </ul>
        <h2>Tarbut School, Science and Mathematics Teacher</h2>
        <ul>
        <li>Jan 2014 – Jan 2015</li>
        <li>Buenos Aires, Argentina</li>
        <li>Designed and implemented creative educational programs</li>
        <li>Coordinated school-wide science and math competitions</li>
        </ul>
        <h2>Evolutionary Studies Laboratory, University of Buenos Aires, Research Assistant</h2>
        <ul>
        <li>Jan 2010 – Jan 2014</li>
        <li>Buenos Aires, Argentina</li>
        <li>Coordinated field expeditions and permit negotiations</li>
        <li>Managed statistical programs and databases</li>
        <li>Trained new laboratory members</li>
        </ul>
        <h1>Projects</h1>
        <h2>Synergistic Discovery and Design (SD2)</h2>
        <ul>
        <li>Jan 2018 – June 2022</li>
        </ul>
        <h2>iBioSim Development</h2>
        <h2>Advanced Machine Learning for Biological Systems</h2>
        <h2>Detecting Fake News</h2>
        <ul>
        <li>Jan 2022 – present</li>
        </ul>
        <h1>Skills</h1>
        <ul>
        <li>Software Engineering: Expert; Keywords: Python, Java, C++, JavaScript, R, Git, GitHub, GitLab, CI/CD, Docker, Kubernetes, Google Cloud, TensorFlow, PyTorch, scikit-learn</li>
        <li>Optimization &amp; Operations Research: Expert; Keywords: CPLEX, Gurobi, Pyomo, Linear Programming, Nonlinear Programming, Integer Programming</li>
        <li>Web Scraping: Intermediate; Keywords: Scrapy, Splash, Selenium</li>
        <li>Probabilistic Modeling: Advanced; Keywords: Gaussian Process Regression, Bayesian Machine Learning</li>
        <li>Data Science &amp; Machine Learning: Expert; Keywords: Artificial Intelligence, Data Analysis, Neural Networks, Physics-Informed Neural Networks, Bayesian Inference, VAR Regression</li>
        <li>Web Development: Expert; Keywords: HTML, Hugo, Jekyll</li>
        <li>Databases: Intermediate; Keywords: SQL</li>
        <li>High Performance Computing: Expert; Keywords: HPC, Parallel Computing, Cluster Computing, MPI, Job Scheduling, Distributed Computing</li>
        </ul>
        <h1>Publications</h1>
        <h2>Evaluating the Contribution of Model Complexity in Predicting Robustness in Synthetic Genetic Circuits (<a href="doi.org/xx.xxxx/acssynbio.xxxxxx">https://doi.org/xx.xxxx/acssynbio.xxxxxx</a>)</h2>
        <ul>
        <li>Jan 2024</li>
        <li>Pedro Fontanarrosa</li>
        <li>ACS Synthetic Biology</li>
        </ul>
        <h2>Synthetic Biology Open Language (SBOL) Version 3.1.0 (<a href="doi.org/xx.xxxx/jib.xxxxxx">https://doi.org/xx.xxxx/jib.xxxxxx</a>)</h2>
        <ul>
        <li>Jan 2023</li>
        <li>Pedro Fontanarrosa</li>
        <li>Journal of Integrative Bioinformatics</li>
        </ul>
        <h1>Awards</h1>
        <ul>
        <li>Fulbright and Argentine Presidential Fellowship in Science &amp; Technology, Date: 2017, Awarder: Argentine President’s Cabinet &amp; U.S. Embassy of Buenos Aires, Summary: Awarded to pursue a master’s degree in the United States starting Fall 2017.</li>
        <li>Research and Communication Excellency Award, Date: 2015, Awarder: Ministry of Science and Technology, Argentinean government, Summary: Recognized for excellence in research and communication under the 'Beca Estímulo' scholarship.</li>
        <li>Beca Estímulo (Encouragement Scholarship), Date: 2011, Awarder: University of Buenos Aires, Summary: Supported research and development tasks in genetics and ecology.</li>
        </ul>
        <h1>Certificates</h1>
        <ul>
        <li>Data Science Bootcamp – THE ERDŐS INSTITUTE (2023); URL: https://www.erdosinstitute.org/certificates/spring-2023/data-science-boot-camp/pedro-fontanarrosa</li>
        <li>Machine Learning A-Z™: AI, Python &amp; R + ChatGPT Bonus [2023] – Udemy (2023); URL: https://www.udemy.com/certificate/UC-099abc5f-7fbc-4674-988d-1c2c96302ec8/</li>
        <li>Optimization with Python: Solve Operations Research Problems – Udemy; URL: https://www.udemy.com/course/optimization-with-python-linear-nonlinear-and-cplex-gurobi/?couponCode=ST10MT30325G2</li>
        <li>Optimization with Python: Complete Pyomo Bootcamp A-Z – Udemy; URL: https://www.udemy.com/course/optimization-with-python-complete-pyomo-bootcamp-a-z/?couponCode=ST10MT30325G2</li>
        <li>AI and Meta-Heuristics (Combinatorial Optimization) Python – Udemy; URL: https://www.udemy.com/course/ai-and-combinatorial-optimization-with-meta-heuristics/?couponCode=ST10MT30325G2</li>
        <li>Modern Web Scraping with Python using Scrapy Splash Selenium – Udemy; URL: https://www.udemy.com/course/web-scraping-in-python-using-scrapy-and-splash/?couponCode=ST10MT30325G2</li>
        <li>Deployment of Machine Learning Models – Udemy; URL: https://www.udemy.com/course/deployment-of-machine-learning-models/?couponCode=ST10MT30325G2</li>
        <li>Pyomo Bootcamp: Python Optimization from Beginner to Advance – Udemy; URL: https://www.udemy.com/course/optimization-in-python/?couponCode=ST10MT30325G2</li>
        <li>Theory of Gaussian Process Regression for Machine Learning – Udemy; URL: https://www.udemy.com/course/gaussian-process-regression-fundamentals-and-application/?couponCode=ST10MT30325G2</li>
        </ul>
        <h1>Languages</h1>
        <ul>
        <li>Spanish: Native</li>
        <li>English: Proficient</li>
        <li>Portuguese: Intermediate</li>
        <li>French: Beginner</li>
        </ul>
        <h1>Volunteer</h1>
        <ul>
        <li>Data-Centric Biological Design &amp; Engineering Interest Group – Organizer, URL: https://data-centric-biodesign-tig.github.io/, Start: 2024-10, End: 2025-05, Summary: Organizes a monthly seminar series under the Alan Turing Institute's banner to leverage AI in advancing biological system engineering. Unites experts from computer science, biology, and engineering to address global challenges in sustainable manufacturing, healthcare innovation, and environmental impact.</li>
        <li>Biohacking BA – Volunteer Organizer, URL: https://www.meetup.com/es/biohackingBA/, Start: 2013-01, End: 2017-01, Summary: Organized talks, workshops, hackathons, and DIY projects to promote innovation in science, engineering, and synthetic biology. Highlights: Organized hackathons for SBOL and FAIR data practices; Coordinated interdisciplinary teams and managed remote collaboration.</li>
        <li>University of Buenos Aires Biology Week – Event Organizer, Start: 2010-01, End: 2012-01, Summary: Coordinated the annual Biology Week to promote science careers among high school students.</li>
        </ul>
        <h1>Interests</h1>
        <h2>Synthetic Biology</h2>
        <h2>Electronics &amp; DIY</h2>
    </article>
</body>

</html>